controller/health.go

```diff
@@ -72,5 +79,8 @@ func setApplicationHealth(resources []managedResource, statuses []appv1.Resource
 	} else {
 		app.Status.ResourceHealthSource = appv1.ResourceHealthLocationAppTree
 	}
+	if savedErr != nil && errCount > 1 {
+		savedErr = fmt.Errorf("encountered this and %d other errors: %w", errCount-1, savedErr)
```

Comment:
Not sure the message is very clear. Does it mean that all X number of error are the same?

Reply:
It does not, the other errors are potentially different and are discarded for brevity.

What about `see applicaton-controller logs for %d other errors; most recent error was: %w`

Reply:
that is a great message. love it!